kube/services/jobs/usersync-job.yaml

```diff
@@ -124,9 +124,9 @@ spec:
               fi
             else
               if [[ -f /mnt/shared/user.yaml ]]; then
-                fence-create sync --sync_from_dbgap $(SYNC_FROM_DBGAP) --projects /var/www/fence/projects.yaml --yaml /mnt/shared/user.yaml
+                fence-create sync --arborist http://arborist-service --sync_from_dbgap $(SYNC_FROM_DBGAP) --projects /var/www/fence/projects.yaml --yaml /mnt/shared/user.yaml
               else
-                fence-create sync --sync_from_dbgap $(SYNC_FROM_DBGAP) --projects /var/www/fence/projects.yaml
+                fence-create sync --arborist http://arborist-service --sync_from_dbgap $(SYNC_FROM_DBGAP) --projects /var/www/fence/projects.yaml
```

Comment:
Would you make the same changes to `usersync-cronjob`?

Reply:
Also - looks like some code needs to be merged into fence?
```
g3kubectl logs useryaml-vxfs9 fence
usage: fence-create [-h] [--path PATH]
                    {create,client-create,client-modify,client-list,client-delete,user-delete,expired-service-account-delete,hmac-create,sync,link-bucket-to-project,google-bucket-create,google-logging-bucket-create,google-manage-keys,google-init,google-manage-account-access,token-create}
                    ...
fence-create: error: unrecognized arguments: --arborist http://arborist-service
```

Reply:
Added usersync-cronjob changes in 12b2919